src/client/utils/destination-location.js

```diff
@@ -79,7 +79,7 @@ export function getParsed () {
     let pathname   = nativeMethods.anchorPathnameGetter.call(resolver);
 
     // NOTE: IE ignores the first '/' symbol in the pathname.
-    if (pathname.indexOf('/') !== 0)
+    if (pathname.charAt(0) !== '/')
```

Comment:
What's the reason for not using `pathname[0]`?

Reply:
I don't know :)
Usually the `charAt` is used in the [testcafe](https://github.com/DevExpress/testcafe/search?q=charAt&unscoped_q=charAt) and [hammerhead](https://github.com/DevExpress/testcafe-hammerhead/search?q=charAt&unscoped_q=charAt).

Reply:
old IE support?